### PR TITLE
Saas 11.4 revert yti

### PR DESCRIPTION
**Pull request by tivisse**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: odoo/addons/base/models/res_users.py [hunk 1]
```diff
@@ -752,6 +777,268 @@ def write(self, values):
...
+    @api.model
+    def default_get(self, fields):
+        group_fields, fields = partition(is_reified_group, fields)
+        fields1 = (fields + ['groups_id']) if group_fields else fields
+        values = super(UsersView, self).default_get(fields1)
+        self._add_reified_groups(group_fields, values)
+        return values
+
+    @api.multi
+    def read(self, fields=None, load='_classic_read'):
```
Comment (tivisse) [inline odoo/addons/base/models/res_users.py]: Reapply patch on translations.

File: odoo/addons/base/models/res_users.py [hunk 2]
```diff
@@ -889,6 +777,295 @@ def write(self, values):
...
+        self.env['ir.actions.actions'].clear_caches()
+        return res
+
+    @api.model
+    def _update_user_groups_view(self):
+        """ Modify the view with xmlid ``base.user_groups_view``, which inherits
+            the user form view, and introduces the reified group fields.
+        """
+        if self._context.get('install_mode'):
+            # use installation/admin language for translatable names in the view
```
Comment (tivisse) [inline odoo/addons/base/models/res_users.py]: @KangOl I can't find the fix about the translations.

Isn't it included just here ?

File: addons/point_of_sale/views/res_users_view.xml [hunk 3]
```diff
@@ -1,14 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <odoo>
-    <record id="res_users_view_form" model="ir.ui.view">
+    <record id="res_users_form_view" model="ir.ui.view">
```
Comment (tivisse) [inline addons/point_of_sale/views/res_users_view.xml]: Check this.

File: addons/auth_signup/views/res_users_views.xml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-        <record id="res_users_view_form" model="ir.ui.view">
+        <record id="res_users_form_view" model="ir.ui.view">
```
Comment (tivisse) [inline addons/auth_signup/views/res_users_views.xml]: Check this.

File: addons/sale/security/sale_security.xml [hunk 5]
```diff
@@ -33,14 +37,23 @@
...
         <field name="name">A warning can be set on a product or a customer (Sale)</field>
+        <field name="category_id" ref="base.module_category_hidden"/>
     </record>
 
     <record id="group_proforma_sales" model="res.groups">
         <field name="name">Pro-forma Invoices</field>
+        <field name="category_id" ref="base.module_category_hidden"/>
+    </record>
+
+    <record id="group_analytic_accounting" model="res.groups">
```
Comment (tivisse) [inline addons/sale/security/sale_security.xml]: Remove this, never used

File: addons/website_forum/__manifest__.py [hunk 6]
```diff
@@ -20,7 +20,7 @@
     'data': [
         'data/forum_data.xml',
         'views/forum.xml',
-        'views/res_users_views.xml',
+        'views/res_users.xml',
```
Comment (tivisse) [inline addons/website_forum/__manifest__.py]: Check this

File: odoo/addons/base/models/res_users.py (hunk 2 above)
Comment (KangOl) [reply odoo/addons/base/models/res_users.py]: This is not a translation fix.
I was talking about commit 79e18ac6afd290a438527d6bfa7fa8d00977a3e2

**Pull request merged**